### PR TITLE
update all.json

### PR DESCRIPTION
**Pull request by ical10**
| Site | Urlscan |
|----|----|
| 62.197.136.191 | https://urlscan.io/result/365e7515-4e10-4abc-b44f-4d767bcd891d/ |
| 4ever-photo.com | https://urlscan.io/result/c7b8817c-071c-4c7a-872b-e35038508400/ |
| outerring-bonus.online | https://urlscan.io/result/c3b7ea10-29e2-4a74-97a2-b924ecc4377d/ |
| sinceredoge-bonus.online | https://urlscan.io/result/14c7a377-0d1b-4d52-8920-86646b326dbe/ |
| naturalfarm-bonus.online | https://urlscan.io/result/a60ff369-74fa-4fde-90ed-e05fc408fb9d/ |
| nftswaps.live | https://urlscan.io/result/9d3ca592-043b-476f-a6f8-366ab0dd39ee/ |
| swappynfts.online | https://urlscan.io/result/84eb0644-07a3-48f8-8db1-f36184f1d1eb/ |
| walletfix.weebly.com | https://urlscan.io/result/3253705e-3fe8-4629-81d4-ca98d09edb2e/ |
| syncvalidatedapps.com | https://urlscan.io/result/97362942-cf33-4064-9265-7dc6d6c04f2e/ |
| blockschainssuppt.site | https://urlscan.io/result/38bee543-9491-4dc4-aad9-f7ee8bba0161/ |
| blockschainexplorer.com | https://urlscan.io/result/17bc9fb3-0d5f-4da3-b796-eea14a6e5450/ | 

62 197 136 191_SITE
4ever-photo com_SITE
outerring-bonus online_SITE
sinceredoge-bonus online_SITE
naturalfarm-bonus online_SITE
nftswaps live_SITE
swappynfts online_SITE
walletfix weebly com_SITE
syncvalidatedapps com_SITE
blockschainssuppt site_SITE
blockschainexplorer com_SITE

### CODE REVIEW

**Comment by ical10** [comment]
added

    "outerring-bonus.online",
    "sinceredoge-bonus.online",
    "naturalfarm-bonus.online",

**Comment by ical10** [comment]
added

    "nftswaps.live",
    "swappynfts.online",

**Comment by ical10** [comment]
added `"walletfix.weebly.com",`

**Comment by ical10** [comment]
added `syncvalidatedapps.com`

File: all.json [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+    "blockschainssuppt.site",
+    "syncvalidatedapps.com",
+    "walletfix.weebly.com",
+    "nftswaps.live",
+    "swappynfts.online",
+    "natural-farm-bonus.online",
+    "outerring-bonus.online",
+    "sinceredoge-bonus.online",
+    "naturalfarm-bonus.online",
+    "62.197.136.191",
```
Comment (jacogr) [inline all.json:14]: @michalisFr  @dubstard This I'm assuming is ok? (I guess it is is a known bad block as well, but will block future legit sites here)

Comment (ical10) [reply all.json:14]: Which one I should remove?

Comment (jacogr) [reply all.json:14]: Not suggesting removing, just asking (out of interest) on the IP.

**Review by jacogr — APPROVED** [review]

**Pull request merged**